base_layer/wallet/tests/transaction_service_tests/service.rs

```diff
@@ -2103,7 +2103,6 @@ async fn manage_multiple_transactions() {
 
     let database_path = temp_dir.path().to_str().unwrap().to_string();
 
-    // TODO: When using a memory type db connection this test fails at `assert_eq!(tx_reply, 3, "Need 3 replies");`
```

Comment:
Has this been fixed?

Reply:
No, but the test does not need a memory db, and if that's still an issue, log an issue. Todo's don't belong in code

Reply:
Maybe then convert it to a normal comment for the next maintainer that may struggle with it? Probably not worth an issue.